docs/en/docs/kairship/workload/deployment.md

```diff
@@ -27,7 +27,7 @@ Follow the steps below to create a multicloud deployment from an image.
     - Specify Clusters: Select the specific cluster to deploy the multicloud workload.
     - Specify Regions: Filter clusters based on the provider/region/availability zone. You can enable all three filters simultaneously.
 
-        - __Exclude Clusters__ : Exclude a specific cluster from the filtering result. The workload will not be deployed to the excluded cluster.
+        - __Exclude Clusters__ : Exclude a specific cluster from the filtering result. The workload will not be deployed to the excluded cluster.If the target cluster is not specified, it will be deployed to all clusters by default.
```

Comment:
```suggestion
        - __Exclude Clusters__ : Exclude a specific cluster from the filtering result. The workload will not be deployed to the excluded cluster. If you do not specify the target cluster, it will be deployed to all clusters by default.
```